exercises/sum-of-multiples/src/test/java/SumOfMultiplesTest.java

```diff
@@ -6,11 +6,11 @@
 public class SumOfMultiplesTest {
 
     @Test
-    public void testSumOfMultiplesOf3and5UpToOne() {
+    public void testNoMultiplesWithinLimit() {
```

Comment:
Just for future reference, our test names don't have to correspond exactly with the canonical data. We only want the test values and test order to be the same. And we want the names to be good descriptive names for the test 🙂 The names you've changed them to seem fine though 🙂

Reply:
The reason why I changed the method names was that the change from version 1.4.0->1.4.1 only included changes in the description for all tests so it is clearer. that's why I thought it would be good to mirror that effort in the test method names.

Reply:
I see, that makes sense 🙂